scripts/calculateRewards/e2eTestRewards.ts

```diff
@@ -0,0 +1,98 @@
+import yargs from 'yargs'
```

Comment:
This whole file is just a stripped-down version of the existing `proofOfImpact.ts` script in the same directory.